test-no-uncommitted-doc-changes.in

```diff
@@ -20,7 +20,7 @@ if [ "$PLONE_VERSION" = "5.0.x" ]; then
     exit 0
 fi
 
-changes=$(git diff --exit-code $DUMPS_DIR)
+changes=$(git diff --ignore-space-at-eol --exit-code $DUMPS_DIR)
```

Comment:
@jaroel nice one. Though, I really wonder why the auto-generated json files have blanks at the end of the line in the frist place. cc @lukasgraf